### ISSUE TITLE
[ig/security] Security Interest Group Charter

### ISSUE THREAD

**Issue by simoneonofri** [w3c/strategy#449]
New charter proposal, reviewers please take note.

# Charter Review

[PROPOSED Security Interest Group Charter:](https://w3c.github.io/charter-drafts/2024/ig-security.html)

[diff from charter template](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2Fcharter-template.html&doc2=https://w3c.github.io/charter-drafts/2024/ig-security.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* New
 - [ ] New WG 
 - [x] New IG
 - If this is a new WG or IG charter request, link to [Advance Notice](https://lists.w3.org/Archives/Public/public-new-work/2024Apr/0001.html), and any issue discussion: 

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, security, and TAG. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach: security groups, privacy groups, security researchers, cryptographers

Known or potential areas of concern: 

Where would charter proponents like to see issues raised? this issue seems fine

Anything else we should think about as we review?

**Comment by simoneonofri** [comment]
Advance Notice: https://lists.w3.org/Archives/Public/public-new-work/2024Apr/0001.html

**Comment by simoneonofri** [comment]
Work on the charter is proceeding. A number of PRs have been prepared.

 https://github.com/w3c/charter-drafts/pulls?q=is%3Apr+is%3Aopen+%5Big%2Fsecurity%5D

**Comment by simoneonofri** [comment]
Updated Charter with some cosmetics and the candidate co-chairs!

https://w3c.github.io/charter-drafts/2024/ig-security.html 

cc'ing @innotommy, Patrick (https://syssec.ethz.ch/people/schaller.html) @jaromil @andrea-dintino

**Comment by ruoxiran** [comment]
no comment or request from APA.

**Comment by himorin** [comment]
- filed [a PR](https://github.com/w3c/charter-drafts/pull/546) for minor points, link targets and list of HR
- participation mentions about agreement on W3C patent policy, which seems restricted condition than patent disclosure section, and seems to be better to refer conditions in disclosure section I suppose... (not sure around legal requirement, although)

**Comment by xfq** [comment]
There seems to be a typo: "Shaller" should be "Schaller"?

**Comment by simoneonofri** [comment]
@xfq thank you noted

**Comment by himorin** [comment]
no comment or request from i18n

**Comment by plehegar** [comment]
PING will talk about this charter on July 18.

**Comment by plehegar** [comment]
One comment from PING:
[[
Maybe worth going into a bit more detail about how this group will coordinate with PING? It does say that one of the group's deliverables is maintained in coordination with TAG and PING, but more information might be useful. Also, under Coordination it says the group's deliverables are "accessibility, internationalization, and privacy" which doesn't seem quite right.
]]

**Comment by simoneonofri** [comment]
Hello everyone,

Thank you for all the comments. Related PR follows:

### Typos

Issue/Comment: https://github.com/w3c/strategy/issues/449#issuecomment-2196012859
Description: I made the corrections indicated and did a general check again.
PR: https://github.com/w3c/charter-drafts/pull/556
Thanks to: @xfq

### Typos

Issue/Comment: https://github.com/w3c/strategy/issues/449#issuecomment-2196012859
Description: I made the corrections indicated and did a general check again.
PR: https://github.com/w3c/charter-drafts/pull/556
Thanks to: @xfq

### Coordination

Issue/Comment: https://github.com/w3c/charter-drafts/issues/547 and https://github.com/w3c/charter-drafts/issues/551
PR: https://github.com/w3c/charter-drafts/pull/560
Description: Added W3C groups
Thanks to: @frivoal and @jyasskin

### Participation

Issue/Comment: https://github.com/w3c/charter-drafts/issues/550
PR: https://github.com/w3c/charter-drafts/pull/558
Description: Clarified that anyone can discuss, to participate need to join. open to IEs with specified expertise
Thanks to: @jyasskin, @chrisn

### Scope and Deliverables (Threat Modeling)

Issue/Comment: https://github.com/w3c/charter-drafts/issues/552
PR: https://github.com/w3c/charter-drafts/pull/557
Description: added emphasis in scope and deliverables
Thanks to: @jyasskin, @jaromil

[cc'ing: @innotommy, @andrea-dintino]

I'll leave them open a week for comments and revisions, then if there are no blockers I'll proceed.

Thank you,

Simone

**Comment by svgeesus** [comment]
Overall looks great. A few very minor points:

- The Note above Motivation section can be deleted. Actually it should be deleted from the template, it serves no purpose.
- Same for the note about charter history
-  [pick a duration within:] one week to 10 working days, Pick one, and remove the todo
- a few leftover todo around the word "Interest"
- you can delete or comment out the charter table rows about extension and rechartering

**Comment by simoneonofri** [comment]
AC review started https://lists.w3.org/Archives/Public/public-new-work/2024Aug/0002.html

**Comment by simoneonofri** [comment]
thx @koalie

**Comment by simoneonofri** [comment]
We included the new deliverable and asked the AC Reps who replied for a review in 2 weeks 

https://lists.w3.org/Archives/Member/member-charters-review/2024Oct/0000.html

**Comment by plehegar** [comment]
[REVIEW by 2024-10-19/20: Proposed changes to the Security Interest Group charter](https://lists.w3.org/Archives/Member/member-charters-review/2024Oct/0000.html)

**Comment by plehegar** [comment]
We received additional feedback during the review of proposed changes.:
- https://github.com/w3c/charter-drafts/pull/613
- https://github.com/w3c/charter-drafts/pull/614
